### ISSUE TITLE
is there a ready-to-go model?

### ISSUE THREAD

**Issue by leolle** [openai/deeptype#34]
Hi, is there anything that I can run without training such a big model?

**Comment by phdowling** [comment]
Seconded, if anyone has a pre-trained model I'd appreciate if you can share / point me to it.

**Comment by aronszanto** [comment]
Agreed. @JonathanRaiman any chance you'd be willing to release the trained model? You'd make the open source community and the environment very happy :)

**Comment by adelra** [comment]
a pre-trained model would be very useful. If you don't have the time, maybe I could be able to help in training and releasing a model.

**Comment by JonathanRaiman** [comment]
Hey @adelra that would be great! I don't have a ready to go model at the moment, but am working on something else in this space where I'll definitely include pretained models

**Comment by adelra** [comment]
@JonathanRaiman  Awesome thanks! Feel free to contact me.

**Comment by ryangawei** [comment]
@JonathanRaiman That would be great! Looking forward to a pre-trained model.

**Comment by phdowling** [comment]
@JonathanRaiman any news on this? If there's a chance you'll release a pre-trained model, that would be highly apprectiated, if it's not gonna happen then a heads up is also appreciated (may try and train my own in that case)

**Comment by Hadjerkhd** [comment]
Any pre-trained model coming soon ? It would be great if someone shares the model he/she trained 
Thanks in advance
